test/config/fileserver_config.json

```diff
@@ -0,0 +1,41 @@
+{"os_image":
+    [ {"osname":"ESXi",
+      "version":"6.0" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/VMware-VMvisor-Installer-6.0.0.update02-3620759.x86_64.iso"
+            },
+      {"osname":"Ubuntu",
+      "version":"16.04" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/ubuntu-14.04.4-server-amd64.iso"
+            },
+      {"osname":"CentOS",
+      "version":"7.0" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/CentOS-7-x86_64-DVD-1511.iso"
+            },
+       {"osname":"Ubuntuserver",
+      "version":"14.04" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/ubuntu-14.04.4-server-amd64.iso"
+            },
+      {"osname":"RedhatEnterpise",
+      "version":"7.2" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/rhel-server-7.2-x86_64-dvd.iso"
+            },
+      {"osname":"openSUSE",
+      "version":"42.1" ,
+      "linktype":"http",
+      "url":"http://10.240.19.193/repo/ISO/openSUSE-Leap-42.1-DVD-x86_64.iso"
+            }
+      ],
+  "microkernel":["http://web.hwimo.lab.emc.com/autotest/staticfile/vmlinuz-3.16.0-25-generic",
+                   "http://web.hwimo.lab.emc.com/autotest/staticfile/discovery.overlay.cpio.gz",
+                   "http://web.hwimo.lab.emc.com/autotest/staticfile/initrd.img-3.16.0-25-generic",
+                   "http://web.hwimo.lab.emc.com/autotest/staticfile/base.trusty.3.16.0-25-generic.squashfs.img"],
+  "usr":"onrack",

```

Comment:
usr/pwd,  should be in the credentials_default.json config file.  access via fitcfg().
can you use the ones already defined there?  (note, you adjust them for your lab defaults.)
Or is this something new for credentials for image-service?  (does not seem to be.)